charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -20,3 +20,164 @@ spec:
   - name: v1alpha1
     served: true
     storage: false
+  preserveUnknownFields: true
```

Comment:
This will be needed in `apiextensions.k8s.io/v1` when pruning becomes the default behavior. We need to set this to `true` while the `MeteringConfig` CRD is still being mapped out.